openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -232,98 +243,87 @@
-        different repositories to gain organization-wide
-        situational awareness is outside the scope
-        of OpenShift configuration.
+        While OpenShift can generate audit data, and securely transport
+        the data to a SIEM, OpenShift does not provide automated mechanisms
+        (e.g. email, text messages) after system events. This functionality
+        would fall to third party software such as Splunk, Elastic, etc.
+        
+        To configure an IdP, see:
+        https://docs.openshift.com/container-platform/latest/authentication/identity_providers/configuring-ldap-identity-provider.html#configuring-ldap-identity-provider
 
 - control_key: AU-6 (4)
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: not applicable
   narrative:
     - text: |
-        Organizational capability to centrally review and analyze
-        audit records from multiple components within the system
-        is outside the scope of OpenShift configuration.
-
-        Note: When AU-4 (1) is enforced OpenShift will be configured
-        to off-load audit records onto a different system
-        or media than the system being audited. In practice this correlates
-        to offloading audit records to a SIEM (e.g. Elastic). However,
-        this control is specifically about centralized review and analysis,
-        and not offloading records, thus AU-6 (4) is marked as not
-        applicable.
+        While OpenShift can generate audit data, and securely transport
+        the data to a SIEM, OpenShift does not provide automated mechanisms
+        (e.g. email, text messages) after system events. This functionality
+        would fall to third party software such as Splunk, Elastic, etc.
+        
+        To configure an IdP, see:
+        https://docs.openshift.com/container-platform/latest/authentication/identity_providers/configuring-ldap-identity-provider.html#configuring-ldap-identity-provider
 
 - control_key: AU-6 (5)
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: not applicable
   narrative:
     - text: |
-        Organizational capability to integrate analysis or audit
-        records with analysis of additional organization-defined
-        data/information collected from other sources to further
-        enhance the ability to identify inappropriate or unusual
-        activity is outside the scope of OpenShift configuration.
-
+        While OpenShift can generate audit data, and securely transport
+        the data to a SIEM, OpenShift does not provide automated mechanisms
+        (e.g. email, text messages) after system events. This functionality
+        would fall to third party software such as Splunk, Elastic, etc.
+        
+        To configure an IdP, see:
+        https://docs.openshift.com/container-platform/latest/authentication/identity_providers/configuring-ldap-identity-provider.html#configuring-ldap-identity-provider
```

Comment:
This text does not address the NIST requirement language at https://nvd.nist.gov/800-53/Rev4/control/Au-6#enhancement-5:

AU-6(5) | AUDIT REVIEW, ANALYSIS, AND REPORTING \| INTEGRATION / SCANNING AND MONITORING CAPABILITIESThe organization integrates analysis of audit records with analysis of [Selection (one or more): vulnerability scanning information; performance data; information system monitoring information; [Assignment: organization-defined data/information collected from other sources]] to further enhance the ability to identify inappropriate or unusual activity.
-- | --

Could you share the thought process behind this change?